src/client.rs

```diff
@@ -329,6 +334,56 @@ impl Client {
         self.list_all_indexes_raw_with(indexes_query).await
     }
 
+    /// Swaps a list of [Index] tuples.
```

Comment:
Maybe we should specified that is only 2 indexes that will be swapped. Because I already find it confusing that you can give a list

Reply:
Since it can be 

```
[
 { indexes: ["a", "b"] },
 { indexes: ["c", "d"] } 
]
```

Its a list of multiple indexes swaps. So in our case a list (as long as you want) of tuples (which means only two)